source/instructors/setupcourses/assignment-settings/groupwork.rst

```diff
@@ -0,0 +1,54 @@
+
+.. image:: /img/groupwork.png
+   :alt: Group Work
+
+- **Paired Programming** - Enable to auto-assign students into groups. Groups can be between 2 and 5 students. See below
+ 
+- **Shared Focus** - Enable to apply Shared Focus.  Shared focus means the "navigator" sees exactly what the "driver" sees. This prevents a student in the "navigator" role from manipulating the view of the driver (changing guide pages, opening other code files, etc) and as the driver moves around in the assignment, the navigator view will change to follow.
+
+- **Number of Students** - Set the number of students to work together as a group (max 5). If the total number of students is greater than an equal division for the groups, extra groups will be created
+
+Managing Groups
+---------------
+
+Once groups have been set up and saved, teachers/instructors can manage the groups going to the **Groups** Tab that will then be shown and **Arrange Groups** button to move students between the groups and/or rename the groups
+
+.. image:: /img/managegroups.png
+   :alt: Managing Groups
+
+Adding New Students
+-------------------
+
+If a new student joins the course after Pair Programming is enabled, they will automatically be added to any existing group that is not already at the maximum number of students. If all groups are already at the maximum number a new group will be created. Teachers/Instructors may then want to review the existing groups so there are no students operating on their own in their own group.
+
+Driver/Navigator
+----------------
+
+Within a group there is a **Driver** who has control of the assignment and **Navigators** who are able to view the work being done by the driver where they can see the drivers cursor location and selections (similar to googleDocs). If the cursor cannot be seen (e.g. the driver is on another page or file in the assignment), clicking on the drivers username in the top panel will take the navigator to their location
+
+To transfer control between users, the navigator can start to type and if the existing driver is not actively working, they will see the Driver toggle switch flip to show they now have control.
+
+If the existing driver is still actively working, the navigator can flip the toggle switch next to "Driver" to request control.
+
+.. image:: /img/driver.png
+   :alt: Driver control
+   
+   
+If the driver is still working they will get a notification asking for control. This will not block the driver from continuing their work if they deny the request and when the are ready to release control, the can toggle the switch to transfer control to the requestor or grant access from the notification pop up if still showing.
```

Comment:
Line 47 the -> they "the can toggle the switch to transfer control"

Reply:
done